### PR TITLE
Update promtail image to support multiarch and bump to 2.4.1

### PR DESCRIPTION
**Pull request by deepsm007**
/cc @yselkowitz

### CODE REVIEW

File: ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-export LOKI_VERSION="2.2.1"
+export LOKI_VERSION="2.4.1"
```
Comment (yselkowitz) [inline ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh:7]: This one updates both loki and promtail in this file, wdyt @vrutkovs ?

**Review by yselkowitz — COMMENTED** [review]
Mostly looks good, but one question:

File: ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh [hunk 2]
```diff
@@ -6,8 +6,8 @@ set -o pipefail
 
 export LOKI_STAGE_ENDPOINT=https://observatorium.api.stage.openshift.com/api/logs/v1/dptp/loki/api/v1
 export LOKI_PROD_ENDPOINT=https://observatorium.api.openshift.com/api/logs/v1/dptp/loki/api/v1
-export PROMTAIL_IMAGE="quay.io/openshift-logging/promtail"
```
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh:9]: I'd rather work with "openshift-logging" to have multiarch version uploaded

File: ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh:7]: I'm fine as long as functionality is not lost, `ipi-install-loki` is most commonly used, so its safe enough

File: ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh (hunk 2 above)
Comment (yselkowitz) [reply ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh:9]: That's currently not an option, but we'll keep it in mind as we continue to work on enablement.

**Review by yselkowitz — APPROVED** [review]
/lgtm
/assign @periklis

**Comment by vrutkovs** [comment]
/approve
/hold
for rehearsals

**Comment by vrutkovs** [comment]
/hold cancel

ci/rehearse/openshift/origin/release-4.9/e2e-aws looks fine

**Pull request merged**